bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetAuxiliaryHandler.java

```diff
@@ -0,0 +1,139 @@
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link OpenWebNetAuxiliaryHandler} is responsible for sending Auxiliary (AUX) commands/messages to the bus
+ * It extends the abstract {@link OpenWebNetThingHandler}.
+ *
+ * NOTICE: Support for handling messages from the bus regarding alarm control has to be implemented
+ *
+ * @author Giovanni Fabiani - Initial contribution
+ *
+ */
+@NonNullByDefault
+public class OpenWebNetAuxiliaryHandler extends OpenWebNetThingHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(OpenWebNetAuxiliaryHandler.class);
+
+    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = OpenWebNetBindingConstants.AUX_SUPPORTED_THING_TYPES;
+
+    public OpenWebNetAuxiliaryHandler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    public void initialize() {
+        super.initialize();
+        OpenWebNetBridgeHandler h = bridgeHandler;
+        if (h != null && h.isBusGateway()) {
+            OpenGateway gw = h.gateway;
+            if (gw != null && gw.isConnected()) {
+                try {
+                    OpenMessage msg = BaseOpenMessage.parse("*#9##");
+                    send(msg);
+                } catch (MalformedFrameException | UnsupportedFrameException | OWNException e) {
+                    logger.warn("Exception while processing command {}: ", e.getMessage());
+                }
+            } else {
+                logger.warn("Gateway is not connected");
+            }
+        } else {
+            logger.warn("Gateway is not available");
+        }
+    }
+
+    /**
+     * Handles Auxiliary command for a channel
+     *
+     * @param channel the channel
+     * @param command the Command
+     */
+    @Override
+    protected void handleChannelCommand(ChannelUID channel, Command command) {
+        logger.debug("handleAuxiliaryCommand() (command={} - channel={})", command, channel);
+        Where w = deviceWhere;
+        if (w != null) {
+            if (channel.getId().equals(CHANNEL_AUX)) {
+                updateStatus(ThingStatus.ONLINE);
+                if (command instanceof StringType) {
+                    try {
+                        if (command.toString().equals(Auxiliary.WhatAuxiliary.ON.name())) {
+                            send(Auxiliary.requestTurnOn(w.value()));
+                        } else if (command.toString().equals(Auxiliary.WhatAuxiliary.OFF.name())) {
+                            send(Auxiliary.requestTurnOff(w.value()));
+                        }
+                    } catch (OWNException e) {
+                        logger.warn("Exception while processing command {}: {}", command, e.getMessage());
+                    }
+                } else {
+                    logger.warn("Unsupported command {} for channel {}", command, channel);
+                }
+            } else {
+                logger.warn("Unsupported ChannelUID {}", channel);
+                updateStatus(ThingStatus.UNKNOWN);
```

Comment:
This is not the receiving of a user command that should change the thing status.
Line to be removed.

Reply:
In my opinion the status of the thing should be set as "UNKWOWN", in  fact in such a case the user could set a wrong channel.

Reply:
Ok tje line 109 has been removed!